packages/material-ui-lab/src/internal/pickers/PickersPopper.tsx

```diff
@@ -95,20 +199,18 @@ const PickersPopper: React.FC<PickerPopperProps & WithStyles<typeof styles>> = (
     }
   }, [open, role]);
 
-  const handleBlur = () => {
```

Comment:
This didn't seem to be covered. Removing for clarity until we find out why this was needed.